internal/servers/common/client_ip.go

```diff
@@ -0,0 +1,63 @@
+package common
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"strings"
+	"sync/atomic"
+
+	"github.com/hashicorp/boundary/internal/errors"
+	"github.com/hashicorp/go-secure-stdlib/listenerutil"
+)
+
+const (
+	RealIpHeader        = "X-Real-Ip"
```

Comment:
Seems this isn't used anywhere right now?

Reply:
ty!